src/server/transaction.cc

```diff
@@ -176,7 +176,8 @@ void Transaction::Shutdown() {
 Transaction::Transaction(const CommandId* cid) : cid_{cid} {
   InitTxTime();
   string_view cmd_name(cid_->name());
-  if (cmd_name == "EXEC" || cmd_name == "EVAL" || cmd_name == "EVALSHA") {
+  if (cmd_name == "EXEC" || cmd_name == "EVAL" || cmd_name == "EVAL_RO" || cmd_name == "EVALSHA" ||
```

Comment:
I hope you checked all places with "EVAL" 🙂 We use some more in main_service.cc, we use abs::StartsWith to be more compact.

Reply:
I'll fix 
https://github.com/dragonflydb/dragonfly/blob/8e3b8ccbe3dff2b222f69209037fae5718eaba6f/src/server/main_service.cc#L1705
this week

Reply:
> I hope you checked all places with "EVAL" 🙂 We use some more in main_service.cc, we use abs::StartsWith to be more compact.

I think so yeah - the `StartsWith` ones are ok as will match `EVAL_RO`? Or do you mean some shouldn't match